### ISSUE TITLE
Davie County, North Carolina, United States: source location changed

### ISSUE THREAD

**Issue by skylarmt** [openaddresses/openaddresses#4958]
The .zip file has changed, it's now available at:  
https://docs.daviecountync.gov/WebLink/Browse.aspx?id=204382&dbid=1&repo=DavieCounty&cr=1

Now it requires right-clicking on "address", selecting "Download Selected Entries", waiting for some AJAX to happen, and finally getting prompted to download from a blob URL.  So it might be a bit of a challenge to automate.

**Comment by ingalls** [comment]
@skylarmt Thanks a ton for bringing this to our attention! Blerg, I hate it when counties decide to go this direction.

@justinelliotmeyers Want to see if you can find any other way to get it? Otherwise we can just cache it

**Comment by skylarmt** [comment]
Maybe this should be a new issue, but I just found NC One Map, a data source for the entire state of North Carolina with GeoJSON, shapefiles, and KML downloads:

https://www.nconemap.gov/datasets/nc-master-address-dataset-2009

Despite the 2009 in the URL, it was last updated in October 2019.  It has a license that seems to require attribution (although that may not be legally binding, since it's government data): "Acknowledgement of products derived form this dataset should cite the following: The source of the NC Master Address Dataset is NC OneMap."

**Comment by ingalls** [comment]
@skylarmt Yup! We're already tracking OneMap here: https://github.com/openaddresses/openaddresses/blob/master/sources/us/nc/statewide.json

Generally we encourage as many overlapping datasets from different sources as possible!

**Comment by justinelliotmeyers** [comment]
locked up in here: https://maps.roktech.net/ROKMAPS_DavieNC/

Ill poke around a bit more to try and find her!
